wasmcloud-wasi-kv/kv-demo/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+wit_bindgen::generate!("kvdemo" in "../wit");
+
+/// Main struct that will implement wasi component imports/exports
+struct Demo;
+
+fn run() -> Result<Option<Vec<u8>>, String> {
+    //print("Running!");
+    println!("Running!");
+    let bucket_name = "bucket";
+    let key = "hello";
+    let value = "wasi";
```

Comment:
Do these really help make code easier to maintain and understand?
If you really want these IMO they should be consts instead, i.e. `const BUCKET: &str = "bucket"` etc

Reply:
these are here so they can be reused in the rest of the *demo execution*, I don't think it matters but will change to constants here.